.github/workflows/e2e_test_for_prs.yml

```diff
@@ -0,0 +1,59 @@
+name: Run e2e for topic branch
+on:
+  pull_request:
+    paths-ignore:
+      - '**.md'
+      - 'cloudformation/*.yml'
```

Comment:
I believe we can also add these files:

- .prettierrc
- .eslintrc.js
- LICENSE
- commitlint.config.js
- github/CODEOWNERS
- github/dependabot.yml

Some folders I've doubts if it's related here or not:
- /scripts
- /assets

Reply:
Workflow marked as required. Will we be able to merge the pr with changes from the ignored list?

Reply:
Re Orkun: yes those could be added as well. I didn't change anything on this section, it has been like that for some time, I only fixed the e2e tests.

Re Ilya: No, merging will be blocked. Pls remember that old e2e tests also had the same problem. I have to write a special logic inside the action to bypass such cases. Thankfully, we did this before so I won't need time to research.

Both of your suggestions are valid, but also a bit out of scope. I will spend one more day to fix these, and would like to merge and move on. If we need more improvements, let's discuss them separately, otherwise this initiative will never reach an end. @Orkuncakilkaya @ilfa